shoop/admin/templates/shoop/admin/products/_edit_base_form.jinja

```diff
@@ -15,7 +15,11 @@
     {{ bs3.field(product_form.barcode) }}
     {{ bs3.field(product_form.gtin) }}
     {{ bs3.field(product_form.category) }}
-    {{ bs3.field(product_form.stock_behavior) }}
```

Comment:
And by the way, instead of emulating the `bs3.field` rendering by hand, why not just modify the `choices` for the field to only include "Unstocked"?